Makefile

```diff
@@ -322,6 +325,49 @@ run-statusq-tests: statusq-tests
 	echo -e "\033[92mRunning:\033[39m StatusQ Unit Tests"
 	$(STATUSQ_BUILD_PATH)/bin/TestStatusQ
 
+
+##
+##	Storybook
+##
+
+STORYBOOK_SOURCE_PATH := storybook
+STORYBOOK_BUILD_PATH := storybook/build
+STORYBOOK_INSTALL_PATH := $(shell pwd)/bin
+STORYBOOK_CMAKE_CACHE := $(STORYBOOK_BUILD_PATH)/CMakeCache.txt
+
+$(STORYBOOK_CMAKE_CACHE): | deps
+	echo -e "\033[92mConfiguring:\033[39m Storybook"
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=$(STORYBOOK_INSTALL_PATH) \
+		-DCMAKE_BUILD_TYPE=$(COMMON_CMAKE_BUILD_TYPE) \
+		-DSTATUSQ_SHADOW_BUILD=OFF \
+		$(COMMON_CMAKE_CONFIG_PARAMS) \
+		-B $(STORYBOOK_BUILD_PATH) \
+		-S $(STORYBOOK_SOURCE_PATH) \
+		-Wno-dev \
+		$(HANDLE_OUTPUT)
+
+storybook-configure: | $(STORYBOOK_CMAKE_CACHE)
+
+storybook-build: | storybook-configure
+	echo -e "\033[92mBuilding:\033[39m Storybook"
+	cmake --build $(STORYBOOK_BUILD_PATH) \
+		--config $(COMMON_CMAKE_BUILD_TYPE) \
+		$(HANDLE_OUTPUT)
+
+storybook-install: | storybook-build
+	echo -e "\033[92mInstalling:\033[39m Storybook"
+	cmake --install $(STORYBOOK_BUILD_PATH) \
+		$(HANDLE_OUTPUT)
+
+run-storybook: storybook-build
+	echo -e "\033[92mRunning:\033[39m Storybook"
+	$(STORYBOOK_BUILD_PATH)/bin/Storybook
```

Comment:
Shouldn't this be:
`$(STORYBOOK_INSTALL_PATH)/Storybook`?

Reply:
Frankly speaking - I don't know from which location it should be launched here in Makefile. And if is target `storybook-install` needed at all...

Reply:
Ok, decided to remove completely. It can be added in the future if really needed.